02_TrackWeb/src/stats.ts

```
@@ -0,0 +1,221 @@
+    innerTable.innerHTML = `
+        <thead>
+            <tr>
+                <th>Titles</th>
+                <th>Active Time</th>
+                <th>Total Time</th>
+            </tr>
+        </thead>
+        <tbody></tbody>
+    `;
+    
+    const innerTableBody = innerTable.querySelector('tbody')!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)